scripts/logbook.py

```diff
@@ -38,76 +38,579 @@
+        "Qubit QC (Library Validation) 4.0" : {
+            "lims_instrument" : {
+                "dest_file" : "Qubit",
+                "details" : {
+                    "udf_Assay" : "",
+                    "udf_Lot no: Qubit kit" : ""
+                }
+            }
+        },
+        "RAD-seq Library Indexing v1.0" : {
+            "lims_instrument" : {
+                "dest_file" : "Bravo",
+                "details" : []
+            },
+            "udf_PCR Machine" : {
+                "dest_file" : "PCR",
+                "details" : []
+            }
+        },
+        "Ribosomal cDNA Depletion and 2nd Amplification (SMARTer Pico) 4.0" : {
+            "lims_instrument" : {
+                "dest_file" : "Bravo",
+                "details" : []
+            },
+            "udf_PCR Machine" : {
+                "dest_file" : "PCR",
+                "details" : []
+            }
+        },
+        "RiboZero depletion" : {
+            "lims_instrument" : {
+                "dest_file" : "Bravo",
+                "details" : []
+            },
+            "udf_PCR Machine" : {
+                "dest_file" : "PCR",
+                "details" : []
+            }
+        },
+        "Setup Workset/Plate" : {
+            "lims_instrument" : {
+                "dest_file" : "Bravo",
+                "details" : []
+            }
+        },
+        "Size Selection (Pippin)" : {
+            "default" : {
+                "dest_file" : "Pippin",
+                "details" : {
+                    "udf_Type: Gel Cassette" : "",
+                    "udf_Lot no: Gel Cassette" : "",
+                    "udf_Type: Marker" : "",
+                    "udf_Lot no: Marker" : "",
+                    "udf_Lot no: Electrophoresis Buffer" : ""
+                }
+            }
+        },
+        "Size Selection (Robocut)" : {
+            "lims_instrument" : {
+                "dest_file" : "Bravo",
+                "details" : []
+            }
+        },
+        "Shear DNA (SS XT) 4.0" : {
+            "lims_instrument" : {
+                "dest_file" : "Covaris",
+                "details" : {
+                    "udf_Lot no: Covaris tube" : ""
+                }
+            },
+            "udf_Instrument Used" : {
+                "dest_file" : "Bravo",
+                "details" : []
+            }
+        },
+        "ThruPlex library amplification" : {
+            "lims_instrument" : {
+                "dest_file" : "Bravo",
+                "details" : []
+            },
+            "udf_PCR Machine" : {
+                "dest_file" : "PCR",
+                "details" : []
+            }
+        },
+        "ThruPlex template preparation and synthesis" : {
+            "lims_instrument" : {
+                "dest_file" : "Bravo",
+                "details" : []
+            },
+            "udf_PCR Machine" : {
+                "dest_file" : "PCR",
+                "details" : []
+            }
+        }
```

Comment:
Nice @chuan-wang now the `dict` is more readable. And as @ewels suggested, you could put this whole `dict` in a `yaml` file and can provide it as mandatory input file while running the script.

Reply:
Forget to mention this yesterday, If you would like to keep this `dict` within the package rather than external input file. you could create a new folder within the repo (for instance say `data`) and put the `yaml/json` file there or even as a python script (for direct `import`)

Reply:
That's what I said above ;) Agreed!